### DISCUSSION TITLE
Coqui-TTS Model test adds subdirectory to model_path when run

### DISCUSSION THREAD

**Discussion by ghost** [coqui-ai/TTS#3983] [General]
I'm in the early stages of setting up my own AI assistant, but after running the below code:
![image](https://github.com/user-attachments/assets/32800c9b-e465-4b4d-bd0f-b350ab2d13ba)

I get the following error:
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:/Pegasus/VoiceModel/model.pth/model.pth'

I have spent hours going over the possible issues with ChatGPT and even started from scratch. Has anyone run into this before? Please let me know what other info you may need.

**Comment by eginhard** [comment]
Yes, this is not very consistent, but if it's a XTTS model, just leave out the `model.pth`
